dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-8.0.feature

```diff
@@ -0,0 +1,27 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.javaee-8.0
+visibility=public
+IBM-App-ForceRestart: install, uninstall
+IBM-ShortName: javaee-8.0
+Subsystem-Version: 8.0.0
+Subsystem-Name: Java EE Full Platform 8.0
+# TODO-EE8 add batch-1.0 and ejb-3.2 once persistence service works w/ JPA 2.2
```

Comment:
Is there an issue open to hold onto this TODO? do we need it for sure or just a really nice to have?

Reply:
yes, this TODO is tracked by the blockers in the parent story to this PR (issue #811)